eng/common/scripts/Prepare-Release.ps1

```diff
@@ -177,8 +177,29 @@ if ($releaseTrackingOnly)
 
 if (Test-Path "Function:SetPackageVersion")
 {
-  SetPackageVersion -PackageName $packageProperties.Name -Version $newVersion -ServiceDirectory $packageProperties.ServiceDirectory -ReleaseDate $releaseDateString `
-    -PackageProperties $packageProperties
+  $changeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $packageProperties.ChangeLogPath
+  $versionOfLatestEntry = $changeLogEntries[0].ReleaseStatus.Trim("()")
```

Comment:
I don't like that we are trimming this outside of the changelog script. Can you please add a helper inside of the changelog operations that can encapsulate this logic there.